modules/ocf_desktop/manifests/pulse.pp

```diff
@@ -1,22 +1,10 @@
 class ocf_desktop::pulse {
+  include ocf::packages::pulse;
 
-  ocf::repackage {
-    # PulseAudio without recommends
-    'pulseaudio':
-      recommends => false;
-    # PulseAudio graphical interfaces
-    [ 'paprefs', 'pavucontrol' ]:
-      recommends => false,
-      require    => Ocf::Repackage['pulseaudio'];
+  # Let Pulse connect over TCP for tunneling audio to the TV
+  exec { 'enable-pulse-tcp':
+    command => 'echo "load-module module-native-protocol-tcp" >> /etc/pulse/default.pa',
```

Comment:
this is also probably better done in `ocf-tv` - we already load `module-zeroconf-discover` on the fly, may as well load this one as well.

Reply:
Sounds good, you can replace `zeroconf-discover` with `native-protocol-tcp`.